LoopCore/LoopSettings.swift

```diff
@@ -8,6 +8,22 @@
 import HealthKit
 import LoopKit
 
+public enum DosingStrategy: Int, CaseIterable {
+    case tempBasalOnly
+    case automaticBolus
+}
+
+public extension DosingStrategy {
+    var title: String {
+        switch self {
+        case .tempBasalOnly:
+            return NSLocalizedString("Temp Basal Only", comment: "Title string for temp basal only dosing strategy")
+        case .automaticBolus:
+            return NSLocalizedString("Automatic Bolus", comment: "Title string for automatic bolus dosing strategy")
```

Comment:
[nit] should this be clarified that this automates both basal and bolus?

Reply:
Again, this is just naming. I think you're right that users should understand this strategy can modulate temp basals (for reducing delivery), but that's more of a documentation/onboarding issue, I think.